api/metadata/permissions.py

```diff
@@ -0,0 +1,77 @@
+from contextlib import suppress
+
+from rest_framework.permissions import IsAuthenticated
+
+from metadata.models import MetadataField
+from organisations.models import Organisation
+
+
+class MetadataFieldPermissions(IsAuthenticated):
+    def has_permission(self, request, view):
+        if not super().has_permission(request, view):
+            return False
+
+        # list is handled by the view
+        if view.action == "list" or view.detail:
```

Comment:
Perhaps not super important but I think we _might_ want to implement a permission check for the `list` action: it would be better for the user to see a 'you do not have permission' vs an empty list, right?

Reply:
As discussed - this view is not nested under organisation